pkg/cmd/cluster.go

```diff
@@ -58,16 +62,21 @@ func doStartCluster(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	opts := cluster.CreateOptions{
+		LoadBalancerHostPort: lbHostPort,
+		ImageRegistryName:    registryName,
+		ImageRegistryPort:    registryPort,
+		WorkerCount:          workerCount,
+	}
+
+	return createCluster(ctx, opts)
+}
+
+func createCluster(ctx context.Context, opts cluster.CreateOptions) error {
 	cm := cluster.NewManager(ClusterConfig)
 
-	if _, err := cm.Exists(ctx); err != nil {
+	if exists, _ := cm.Exists(ctx); !exists {
```

Comment:
Should we report on this error, or maybe retry?

Reply:
See below.